src/components/containerApp/ContainerApp.tsx

```diff
@@ -1,11 +1,18 @@
 import { PropsWithChildren } from 'react';
 import { SafeAreaView, View } from 'react-native';
 
+import { Loader } from 'components/loader/Loader';
+
 import { stylesContainerApp as styles } from './ContainerApp.styles';
 
-export const ContainerApp = ({ children }: PropsWithChildren) => {
+interface Props {
+    isLoading: boolean;
```

Comment:
looks like we need `isData` here and condition for `notFound`

Reply:
fixed